test/spec/controllers/tasksCtrlSpec.js

```diff
@@ -30,11 +30,89 @@ describe('Tasks Controller', function() {
   });
 
   describe('editTask', function() {
-    it('is Tasks.editTask', function() {
-      inject(function(Tasks) {
-        expect(scope.editTask).to.eql(Tasks.editTask);
-      });
+
+    var task;
+
+    beforeEach(function(){
+      task = specHelper.newTask();
+    });
+
+    it('toggles the _editing property', function() {
+      scope.editTask(task, user);
+      expect(task._editing).to.eql(true);
+    });
+
+    it('sets _tags to true by default', function() {
+      scope.editTask(task, user);
+
+      expect(task._tags).to.eql(true);
+    });
+
+    it('sets _tags to false if preference for collapsed tags is turned on', function() {
+      user.preferences.tagsCollapsed = true;
+      scope.editTask(task, user);
+
+      expect(task._tags).to.eql(false);
+    });
+
+    it('sets _advanced to true by default', function(){
+      user.preferences.advancedCollapsed = true;
```

Comment:
You should update the `specHelper.newUser()` function to include this default setting.  [Here](https://github.com/HabitRPG/habitrpg/pull/6943/files#diff-ecd2e89a7b19d9ac70c6d589bb181bd9R39) is an example of where this has been done before.